src/modules/recipies/SearchBar/index.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react"
+import './styles.css';
+
+export const SearchBar = ({ onSearch }) => {
+    // TODO: add input to search recipes>
+    const [searchLetter, setSearchLetter] = useState('');
+
+    const handleInputChange = (e) => {
+        setSearchLetter(e.target.value);
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if (searchLetter.length === 1) {
+            onSearch(searchLetter);
+        }
+    };
+
+    return (
+        <form className="search-content" onSubmit={handleSubmit}>
+            <h1 className="line-hero">Search for recipes</h1>
+            <input
+                className="input"
+                type="text"
+                placeholder="Egg Benedict..."
```

Comment:
The search only allows the user to enter 1 letter but placeholder text suggests to add a longer word which is misleading. Add an example of what is actually expected from the user instead.